scrum-board-system/src/components/userstories/UserStory.js

```diff
@@ -1,23 +1,138 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Button, Modal } from 'react-bootstrap'
+const Sprint = ({ sprint }) => {
+  return (
+    <div className="form-group row">
+      <label htmlFor="SprintId" className="col-sm-2 col-form-label">Sprint</label>
+      <div className="col-sm-2">
+        {sprint &&
+          <a id="SprintId" className="badge badge-pill badge-primary p-2"> {sprint.name} </a>
+        }
+      </div>
+    </div>
+  )
+}
 
-export const UserStory = ({ id, title, description, weight, currentProjectId,deleteUserStory}) => {
-    return (
-        <li className={"list-group-item d-flex justify-content-between"} id={id}>
-            {title}
-            <div >
-                <span className="badge badge-primary badge-pill mr-3">{weight}</span>
-                <span className="mr-2" onClick={()=>deleteUserStory(currentProjectId,id)}>
-                    <FontAwesomeIcon icon="trash" />
-                </span>
-                <span >
-                    <FontAwesomeIcon icon="edit" className="invisible"/>
-                </span>
+const LabelContainer = ({ labels }) => {
+  return (
+    <div className="form-group row">
+      <label htmlFor="SprintId" className="col-sm-2 col-form-label">Labels</label>
+      <div className="col-sm-2 center">
+        {labels && labels.map(label => {
+          return (
+            <a key={label.id} className="badge badge-pill badge-info p-2"> {label.description} </a>
+          )
+        })}
+      </div>
+    </div>
+  )
+}
+
+const Task = ({ task }) => {
+  return (
+    <div key={task.id} className="input-group mb-3">
+      <div className="input-group-prepend">
+        <div className="input-group-text">
+          <input type="checkbox" aria-label="Checkbox for following text input" value={task.done} />
+        </div>
+      </div>
+      <input type="text" className="form-control" aria-label="Text input with checkbox" value={task.description} disabled />
+    </div>
+  )
+}
+
+const TasksContainer = ({ tasks }) => {
+  return (
+    <div className="form-group row align-items-center">
+      <div className="col-sm-2">Tasks</div>
+      {tasks && tasks.map(task => (
+        <Task key={task.id} task={task} />
+      ))}
+    </div>
+  )
+}
+
+const UserStory = (props) => {
+  const { userStory, deleteUserstory, projectId, setUserStoryEditHandler } = props
+  const { title, description, weight, labels, tasks, sprint, id } = userStory
+  const [showModal, setShowModal] = useState(false)
 
+  const handleShowModal = () => setShowModal(true)
+  const handleClose = () => setShowModal(false)
+
+  return (
+    <div className="card">
```

Comment:
Could be good to change these elements to Bootstrap react components

Reply:
Done!